sklearn/cluster/k_means_.py

```diff
@@ -724,7 +734,7 @@ class KMeans(BaseEstimator, ClusterMixin, TransformerMixin):
 
     def __init__(self, n_clusters=8, init='k-means++', n_init=10, max_iter=300,
                  tol=1e-4, precompute_distances='auto',
-                 verbose=0, random_state=None, copy_x=True, n_jobs=1):
+                 verbose=False, random_state=None, copy_x=True, n_jobs=1):
```

Comment:
On 01/13/2015 03:02 PM, ragv wrote:
>
> In sklearn/cluster/k_means_.py 
> <https://github.com/scikit-learn/scikit-learn/pull/4023#discussion-diff-22890091>:
>
> > @@ -724,7 +734,7 @@ class KMeans(BaseEstimator, ClusterMixin, TransformerMixin):
> >
> >      def __init__(self, n_clusters=8, init='k-means++', n_init=10, max_iter=300,
> >                   tol=1e-4, precompute_distances='auto',
> > -                 verbose=0, random_state=None, copy_x=True, n_jobs=1):
> > +                 verbose=False, random_state=None, copy_x=True, n_jobs=1):
>
> oh... I just described its type as |{1, 0}|... I assume that is wrong 
> then?
>
I don't know how it is handled in this function. In general it can have 
more values. Not all instances of verbose use values other than 0 and 1.